### DISCUSSION TITLE
Estou querendo iniciar em React JS, o curso GoStack vale a pena?

### DISCUSSION THREAD

**Discussion by rsjayme** [frontendbr/forum#1743] [Discussão]
Boa noite!
Alguém já fez o curso GoStack  da Rocketseat por aqui? Gostaria de opiniões sobre o curso, fiz a semana omnistack e foi meu primeiro contato com React JS e React Native, e estou querendo aprofundar no assunto.
Recebi também remcomendações do curso DevPleno (que por sinal está mais barato), porém não sei se está bem atualizado.

Desde já agradeço!

**Comment by theandersonn** [comment]
@rsjayme Tem essa issue aqui com algumas opiniões de quem já fez. https://github.com/frontendbr/forum/issues/1512

Semana passada tbm acompanhei a omnistack, consegui chegar no final e curti. Decidi participar do Bootcamp e fiz minha inscrição hoje. 

Tem o discord da Rockeseat com uma galera que já fez e outros ainda farão.

Tbm me inscrevi no curso DevPleno, as explicações são boas, mas como um todo não curti, precisa ter uma organização melhor nos módulos e na sequencia da ementa. Precisa dar uma refinada.

**Comment by wilsonneto-swe** [comment]
Fala @rsjayme , 
para mim valeu muito, mas antes dele fiz o Dev Pleno - React JS, que teve um custo benefício melhor até, e após ele eu já comecei a pegar demandas de front também na empresa, então comprei o Go Stack para ter uma visão geral e me afiar em ReactJS, Node e RN.

O link para o curso do Dev Pleno é: [http://bit.ly/reactjs-dev-pleno](http://bit.ly/reactjs-dev-pleno) 

Dê uma olhada nele também, muito bom mesmo.

Abraço!

----

Wilson Gomes
Linkedin: [https://www.linkedin.com/in/wilsonnetobr/](https://www.linkedin.com/in/wilsonnetobr/)

**Comment by KNan-L** [comment]
@wilsonneto-dev Oi, eu to no mesmo barco, to terminando o lauchbase e gostaria de fazer o gostack mas sem condições pelo custo, ai to de olho no devpleno que ta mais em conta, já que você fez os dois, você poderia dizer a diferença entre um e outro?

**Comment by wilsonneto-swe** [comment]
Achei ambos ótimos, o dev pleno até passa mais por teoria, mas são bem mão na massa mesmo. Gosto bastante desta abordagem. Mas o dev pleno reactjs foca bem apenas no reactjs, já o devpleno fullstack foca em todas as plataformas, mas eu hoje tendo feito, começaria pelo devpleno focado em reactjs, o mercado é bem maior e aceita melhor entrantes do que o mercado backend ou mobile, focaria em ficar pró no front e  conseguir entrar no mercado, dai juntaria e partiria oara o go stack para ir a fundo no mobile e node tbm. É o que eu seguiria, pq é melhor focar em um apenas no começo para conseguir evoluir bem, e entrando no mercado e lidando com situações do dia a dia facilita muito  a vida.

Boa sorte e abraço!

---

Wilson Gomes
Linkedin: [https://www.linkedin.com/in/wilsonnetobr/](https://www.linkedin.com/in/wilsonnetobr/)

**Comment by KNan-L** [comment]
@wilsonneto-dev muito obrigada pela resposta, era exatamente o tipo de explicação que eu tava procurando!
